src/commands/targeting/update.ts

```diff
@@ -0,0 +1,120 @@
+import { Args, Flags } from '@oclif/core'
+import UpdateCommand from '../updateCommand'
+import { EnvironmentPromptResult, FeaturePromptResult, environmentPrompt, featurePrompt } from '../../ui/prompts'
+import inquirer from '../../ui/autocomplete'
+import {
+    UpdateTargetingParamsInput,
+    fetchTargetingForFeature,
+    updateFeatureConfigForEnvironment
+} from '../../api/targeting'
+import { TargetingListOptions } from '../../ui/prompts/listPrompts/targetingListPrompt'
+import { FeatureConfig, UpdateTargetParams } from '../../api/schemas'
+import Base from '../base'
+import { validateSync } from 'class-validator'
+import { plainToClass } from 'class-transformer'
+import { reportValidationErrors } from '../../utils/reportValidationErrors'
+
+export default class UpdateTargeting extends Base {
+    static hidden = false
+    static description = 'Update Targeting rules'
+
+    static args = {
+        feature: Args.string({
+            description: 'The Feature for the Targeting Rule.'
+        }),
+        environment: Args.string({
+            description: 'The Environment where the Targeting Rule will be updated.'
+        }),
+    }
+
+    static flags = {
+        ...UpdateCommand.flags,
+        targets: Flags.string({
+            description: 'List of trageting rules'
```

Comment:
```suggestion
            description: 'List of targeting rules'
```

Reply:
We could possibly reference the API docs for the JSON schema expected here? https://docs.devcycle.com/management-api/#tag/Feature-Configurations/operation/FeatureConfigsController_update

Reply:
it might be a little confusing to reference the API docs since we have different display names for properties. e.g. `name` references the audience name, `serve` is the `_variation` field of the distribution property, and definition is the `filters.filters` property of audience. If we reference the docs then we'd also have to explain the mapping of display names to actual properties. Also we might end up with a dead link in some versions of the CLI if the docs API URL ever changes. But I'm totally fine with putting it in if what I just said isn't that big of a deal